### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by saurav-c**
## Describe your changes and why you are making these changes
Prepares for release 0.3.0

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+    now has support for invoking LLMs with a single API call and comes with
+    pre-built Docker images optimized for executing LLMs on Kubernetes. The
+    `llm_op` API supports both ad hoc execution, as pictured below, as well as
+    batch execution over a list of inputs or a Pandas `Series`. See our
+    documentation for more details. **TODO: link the word documentation**
+    ```python
+    import aqueduct as aq
+
+    llm_op = aq.llm_op('vicuna-7b', engine='k8s')
+    llm_op('What is the best LLM?')
```
Comment (cw75) [inline CHANGELOG.md:17]: Let's change the variable name to sth else as it conflicts with the module name...
Let's name it `vicuna`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+* Reorganizes integrations around the concept of resources. Resources are any
+    external tool, system, or API that Aqueduct can connect to; existing data
+    and compute integrations are automatically converted into resources. A
+    container registry resource is added in this release, and future releases
+    will introduce new resource types. The recommended SDK API for accessing
+    resources is now `client.resource`, with `client.integration` slated to
+    deprecated in a future release.
+* Allows users to specify a custom Docker image when running an Aqueduct
+    operator on Kubernetes. The Docker image is required to have the Aqueduct
+    execturo scaffolding installed; for more details, please see our
```
Comment (cw75) [inline CHANGELOG.md:29]: typo: executor?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+    documentation here. **TODO: link documentation**
```
Comment (cw75) [inline CHANGELOG.md:30]: I'm writing the doc this evening.

**Review by cw75 — APPROVED** [review]
@saurav-c Unblocking but left a few comments on the release note.

**Comment by vsreekanti** [comment]
What's the situation with this PR?

**Comment by saurav-c** [comment]
Closing because we released 0.3.1 instead

**Pull request closed without merging**